src/main/java/com/eprosima/fastcdr/idl/templates/TypesHeader.stg

```diff
@@ -189,6 +201,20 @@ public:
 };
 >>
 
+operationRetType(operation) ::= <%
```

Comment:
utility templates, not idl-grammar template, are located after line `/***** Utils *****/`.

Reply:
36a509b